projects/plugins/jetpack/modules/carousel/jetpack-carousel.js

```diff
@@ -38,35 +38,19 @@
 		function getAverageColor( imgEl ) {
 			var canvas = document.createElement( 'canvas' ),
 				context = canvas.getContext && canvas.getContext( '2d' ),
-				imgData,
-				width,
-				height,
-				length,
-				rgb = { r: 0, g: 0, b: 0 },
-				count = 0;
+				rgb = { r: 0, g: 0, b: 0 };
```

Comment:
It's just occurred to me that this is not a good default rgb if the carousel is in Light Mode; the images will then get a gray background because of the alpha gradient. We could add a check for the `jp-carousel-light` class and return white 🤔